### PR TITLE
Fix incorrect example (pressable -> focusable)

### PR DESCRIPTION
**Pull request by terrierscript**
I found incorrect doc.

Codesandbox sample is incorrect too but I can't fix it.
(If need codexandbox too, please close this issue)

### CODE REVIEW

**Comment by Andarist** [comment]
rlly2kryrn is the codesandbox's ID of the focusable example

File: packages/popmotion-pose/docs/examples/react/event-focus.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ To animate an element on focus, set `focusable: true` and a `focus` prop.
 
 ```javascript
 const config = {
-  pressable: true,
+  focusable: true,
   init: { scale: 1 },
   press: { scale: 0.8 }
```
Comment (InventingWithMonster) [inline packages/popmotion-pose/docs/examples/react/event-focus.md]: Should be `focus`

Comment (terrierscript) [reply packages/popmotion-pose/docs/examples/react/event-focus.md]: Thank you!
I copy code from codesandbox

**Comment by InventingWithMonster** [comment]
thanks!

**Pull request merged**